utils/gyb.py

```diff
@@ -304,7 +309,7 @@ def splitGybLines(sourceLines):
     dedents = 0
     try:
         for tokenKind, tokenText, tokenStart, (tokenEndLine, tokenEndCol), lineText \
-            in tokenize.generate_tokens(sourceLines.__iter__().next):
+            in tokenize.generate_tokens(partial(next, sourceLines.__iter__())):
```

Comment:
Blah, you're right. Forgot that `generate_tokens` wants a callable. You might be able to get by with:

```
tokenize.generate_tokens(sourceLinesIter.next)
```

...since `next` is a method on an iterator, though I'm not entirely sure. Otherwise, I guess just stick with a `lambda` or `partial`